### PR TITLE
[bitnami/redis] Fixing the Redis VIB workflow

### PR DESCRIPTION
**Pull request by superaleks**
Signed-off-by: Aleksandra Lukic <alukic@vmware.com>

This is just a test of the Redis workflow. I'm triggering it by making a change in the Redis README.md

### CODE REVIEW

**Comment by beltran-rubo** [comment]
It requires to update the workflows/vib-redis.yaml to point to this new file

**Review by gdelgadot — DISMISSED** [review]

File: bitnami/redis/README.md [hunk 1]
```diff
@@ -544,6 +544,7 @@ useExternalDNS:
     ttl: 10
 ```
 
+
```
Comment (recena) [inline bitnami/redis/README.md:547]: Unrelated change.

Comment (superaleks) [reply bitnami/redis/README.md:547]: Yes, I added a new line in redis README.md just to make sure the Redis workflow gets triggered. This will not enter the final round of changes. It is still in testing.

Comment (recena) [reply bitnami/redis/README.md:547]: `$ git commit --allow-empty -m "Trigger pipeline"`

**Review by beltran-rubo — APPROVED** [review]
